api/src/main/java/uk/gov/dhsc/htbhf/claimant/exception/CardClientException.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.dhsc.htbhf.claimant.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.RestClientException;
+
+/**
+ * Exception used when calling the card service from the {@link uk.gov.dhsc.htbhf.claimant.service.CardClient}.
+ */
+public class CardClientException extends RuntimeException {
+
+    private static final long serialVersionUID = 1L;
+
+    public CardClientException(HttpStatus httpStatus) {
+        super("Response code from card service was not CREATED, received: " + httpStatus.value());
```

Comment:
Minor point but you're mixing HttpStatus codes and names here, can we have either (the same) or both please?